src/test/java/org/mockito/internal/handler/InvocationNotifierHandlerTest.java

```diff
@@ -121,11 +120,9 @@ public void should_report_listener_exception() throws Throwable {
     public void should_delegate_all_MockHandlerInterface_to_the_parameterized_MockHandler() throws Exception {
         notifier.getInvocationContainer();
         notifier.getMockSettings();
-        notifier.setAnswersForStubbing(new ArrayList<Answer<?>>());
 
         verify(mockHandler).getInvocationContainer();
         verify(mockHandler).getMockSettings();
-        verify(mockHandler).setAnswersForStubbing(ArgumentMatchers.<Answer<?>>anyList());
```

Comment:
Thank you for review! Absolutely let's avoid breaking changes!

Why do you consider this a breaking change? Those changes are all in "org.mockito.internal" components. Is it because VerificationData API already leaks some internal APIs? :)